index.html

```diff
@@ -284,14 +267,14 @@ <h5 class="ets" >ETS CONNECT</h5>
                  <img src="images/angellist.PNG" alt="abc">
              </div>
              
-  <button class="sec3rbutton sec3rlink4 ">Get my resume</button>
+  <button type="submit" class="sec3rbutton sec3rlink4 ">Get my resume</button>
 
 
 
 
   <div class="sbddd">
     
-     <label for="cars"> <h3>Languages</label>
+    <h3><label for="cars"> Languages</label></h3>

```

Comment:
- [ ] Please fix the `font` , `margins` , `button` styles to match the Figma template .  To pass this milestone you need to make your design similar to the Figma template . Cheers 👍 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/178113628-6f9bf5d5-c757-44bf-b8f5-d4894df9b418.png)  |  ![image](https://user-images.githubusercontent.com/60618877/178113765-b094be2e-42c7-41f0-ba6c-aabf7159d059.png)